recipes/ommprotocol/meta.yaml

```
@@ -0,0 +1,58 @@
+{% set name = "ommprotocol" %}
+{% set version = "0.1.13" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/insilichem/ommprotocol/archive/v{{ version }}.tar.gz
+  sha256: a58dde3c860a72a3a3ea4590eee2ef2ce3197ce03ac6512c8d537dfc64ef311b
+
+build:
+  noarch: python
+  script: {{ PYTHON }} -m pip install . -vv
+  number: 0
+  entry_points:
+    - ommprotocol = ommprotocol:run_protocol
+    - ommanalyze = ommprotocol.analyze:main
+    - ommstate2pdb = ommprotocol:state_to_pdb
+    - ommexportframe = ommprotocol:export_frame
+
+requirements:
+  host:
+    - python >=3.6
+    - pip
+  run:
+    - python >=3.6
+    - openmm
```

OMMProtocol still expects `simtk` as a namespace for the imports, but this was deprecated in openmm 7.6, so let's add an upper bound to exclude these.

```suggestion
    - openmm <7.6.0a
```